.circleci/config.yml

```diff
@@ -0,0 +1,55 @@
+version: 2
+jobs:
+  build:
+    machine: true
+    working_directory: ~/speaktome
+    steps:
+      - run:
+          name: Install essential packages
+          command: |
+            sudo apt-get update
+            sudo apt-get install -y ca-certificates curl build-essential make git
+
+      - checkout
+
+      - run:
+          name: Create version.json
+          command: |
+            # create a version.json per https://github.com/mozilla-services/Dockerflow/blob/master/docs/version_object.md
+            printf '{"commit":"%s","version":"%s","source":"https://github.com/%s/%s","build":"%s"}\n' \
+            "$CIRCLE_SHA1" \
+            "$CIRCLE_TAG" \
+            "$CIRCLE_PROJECT_USERNAME" \
+            "$CIRCLE_PROJECT_REPONAME" \
+            "$CIRCLE_BUILD_URL" > ~/speaktome/version.json
```

Comment:
why are you referencing this file using the full path instead of putting it in . (since docker is built on .)